### PR TITLE
Switch to simpler stripes setup with webpackOverrides

### PR DESCRIPTION
**Pull request by cherewaty**
Updates to `stripes-core` streamlining done here: https://github.com/folio-org/stripes-core/pull/85

### CODE REVIEW

File: demo/stripes.js [hunk 1]
```diff
@@ -1,47 +1,27 @@
...
-
 // Display webpack output to the console
 function processStats(err, stats) {
   if (err) {
     console.error(err); // eslint-disable-line no-console
   }
   console.log(stats.toString({ // eslint-disable-line no-console
     chunks: false,
-    colors: true
+    colors: true,
```
Comment (sadatay) [inline demo/stripes.js]: this isn't an eslint issue?  thought we had no trailing commas enforced

Comment (cherewaty) [reply demo/stripes.js]: Nope, we just have `comma-dangle` turned off completely, so our repo has no opinion on the matter. I copied and pasted this from `stripes-core`, which does require dangling commas and enforces it.

Comment (cowboyd) [reply demo/stripes.js]: 🎵  Mr. Code Dangles.... 🎵

Comment (sadatay) [reply demo/stripes.js]: 🆒  thanks for the explanation

**Pull request merged**